### PR TITLE
Added initial rollrelease script to perform per-ring steps

### PR DESCRIPTION
**Pull request by jtpetty**
Also, converted mkrelease to use octokit
Also, fixed strings in mkrelease

### CODE REVIEW

File: release/mkrelease.js [hunk 1]
```diff
@@ -94,27 +101,42 @@ async function fetchPRsSinceLastReleaseAndEditReleaseNotes(newRelease, callback)
...
-    if (body.published_at === undefined)
+        var lastReleaseDate = releaseInfo.data.published_at;
+        console.log(`Fetching PRs merged since ${lastReleaseDate}`);
+        try
+        {
+            var results = await octokit.search.issuesAndPullRequests({
+                q:`type:pr+is:merged+repo:${owner}/${repo}+merged:>=${lastReleaseDate}`,
+                order: 'asc',
+                sort: 'created'
+            })
```
Comment (damccorm) [inline release/mkrelease.js]: Not a huge deal if this is hard, but is there a way to filter on branch? This only matters if PRs are done on non-master branches - that _could_ happen if we need to do something like correct release notes or if people are working on a feature together not on master. Idk if we'll ever actually do either of those

File: release/rollrelease.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    var releaseId = releaseInfo.data.id;
+
+    // TODO: Add other actions to take when rolling agent to specific rings
+    //   Some ideas:
+    //      - Update release body
+    //      - Post to Slack Channel
+
+    if (opt.options.stage.toLowerCase() === 'ring 5')
```
Comment (damccorm) [inline release/rollrelease.js]: Is this identical to the variable we populate in our config change release?

Comment (jtpetty) [reply release/rollrelease.js]: yeah, we should be able to do --stage $STAGE_NAME from the config change and it has the value 'Ring 5'. At least that's what I remembered and what my intention was.

File: release/mkrelease.js (hunk 1 above)
Comment (jtpetty) [reply release/mkrelease.js]: good point,  let me check

**Pull request merged**